### PR TITLE
Bootstrap AutoInitialize - Fix typos due to bad merge

### PR DESCRIPTION
**Pull request by DrusTheAxe**

### CODE REVIEW

File: dev/WindowsAppRuntime_BootstrapDLL/MddBootstrapAutoInitializer.cpp [hunk 1]
```diff
@@ -8,16 +8,18 @@
...
-    !defined(MICROSOFT_WINDOWSAPPSDK_BOOTSTRAP_AUTO_INITIALIZE_OPTIONS_ONNOMATCH_SHOWUI) && \
-    !defined(MICROSOFT_WINDOWSAPPSDK_BOOTSTRAP_AUTO_INITIALIZE_OPTIONS_ONPACKAGEIDENTITY_NOOP)
+#if !defined(MICROSOFT_WINDOWSAPPSDK_BOOTSTRAP_AUTO_INITIALIZE_OPTIONS_DEFAULT)
+// Default isn't defined. Define it if no options are defined
+#if defined(MICROSOFT_WINDOWSAPPSDK_BOOTSTRAP_AUTO_INITIALIZE_OPTIONS_NONE)
+#elif defined(MICROSOFT_WINDOWSAPPSDK_BOOTSTRAP_AUTO_INITIALIZE_OPTIONS_ONERROR_DEBUGBREAK)
+#elif defined(MICROSOFT_WINDOWSAPPSDK_BOOTSTRAP_AUTO_INITIALIZE_OPTIONS_ONERROR_DEBUGBREAK_IFDEBUGGERATTACHED)
+#elif defined(MICROSOFT_WINDOWSAPPSDK_BOOTSTRAP_AUTO_INITIALIZE_OPTIONS_ONERROR_FAILFAST)
+#elif defined(MICROSOFT_WINDOWSAPPSDK_BOOTSTRAP_AUTO_INITIALIZE_OPTIONS_ONNOMATCH_SHOWUI)
+#else defined(MICROSOFT_WINDOWSAPPSDK_BOOTSTRAP_AUTO_INITIALIZE_OPTIONS_ONPACKAGEIDENTITY_NOOP)
```
Comment (loneursid) [inline dev/WindowsAppRuntime_BootstrapDLL/MddBootstrapAutoInitializer.cpp:18]: I would have expected:
#elif defined(MICROSOFT_WINDOWSAPPSDK_BOOTSTRAP_AUTO_INITIALIZE_OPTIONS_ONPACKAGEIDENTITY_NOOP)
#else
#define MICROSOFT_WINDOWSAPPSDK_BOOTSTRAP_AUTO_INITIALIZE_OPTIONS_DEFAULT
#endif

but maybe I'm missing something

**Review by loneursid — COMMENTED** [review]
Left a comment / question

File: dev/WindowsAppRuntime_BootstrapDLL/MddBootstrapAutoInitializer.cpp (hunk 1 above)
Comment (DrusTheAxe) [reply dev/WindowsAppRuntime_BootstrapDLL/MddBootstrapAutoInitializer.cpp:18]: Doh! You're right. A little too much copy/paste. Fixing. Thanks!

**Review by kythant — APPROVED** [review]

**Pull request merged**